model_cards/Primer/bart-squad2/README.md

```diff
@@ -0,0 +1,63 @@
+---
+language: "en"
+---
+
+# BART-Squad2
+
+## Model description
+
+BART for extractive (span-based) question answering, trained on Squad 2.0.
+
+F1 score of 87.4.
+
+## Intended uses & limitations
+
+Unfortunately, the Huggingface auto-inference API won't run this model, so if you're attempting to try it through the input box above and it complains, don't be discouraged!
```

Comment:
I removed the `summarization` key in your model's config.json's `"task_specific_params"` so it's now properly loaded as a QA pipeline.

cc @sshleifer @mfuntowicz

Reply:
<img width="695" alt="Screenshot 2020-09-01 at 23 52 03" src="https://user-images.githubusercontent.com/326577/91909795-df514800-ec7b-11ea-9295-19df1c80f717.png">